src/frontends/paddle/src/op/group_norm.cpp

```diff
@@ -0,0 +1,57 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/paddle/node_context.hpp"
+#include "openvino/frontend/paddle/visibility.hpp"
+#include "openvino/opsets/opset6.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+NamedOutputs group_norm(const NodeContext& node) {
+    auto data = node.get_input("X");
+    auto dtype = data.get_element_type();
+    auto num_groups = node.get_attribute<int32_t>("Groups");
+    auto epsilon = node.get_attribute<float>("Epsilon");
+    auto data_layout = node.get_attribute<std::string>("Data_layout");
```

Comment:
better use the default value for 'get_attribute()' function in case it throw error for models which doesn't set this attributes

Reply:
I faced this problem and I don't know how to fix it. @OpenVINO-dev-contest 
![image](https://user-images.githubusercontent.com/69072522/181695635-f43b50ee-c2af-4281-b173-f910b72f4081.png)
I think this led to the error described below
![image](https://user-images.githubusercontent.com/69072522/181702049-6072aa30-8415-4c39-884e-a6c194235bf1.png)
1. I can't find the error log called "unable_build_paddle_models.txt".
2. It's made by wrong code? or compilation mistaken? I want to figure out the potential factors.